macosx/HBUtilities.m

```diff
@@ -167,17 +167,39 @@ + (NSURL *)mediaURLFromURL:(NSURL *)URL
     return mediaURL;
 }
 
++ (NSDate *)tryGetReleaseDate:(HBTitle *)title
```

Comment:
"tryGet" is not usually used for method names in Objective-c, it the returned value can be nil, just annotate it with "nullable".

Reply:
- [ ] annotate method with null and reconsider name